.github/workflows/RollPyTorch.yml

```diff
@@ -1,7 +1,6 @@
 name: Roll PyTorch
 
-on:
-  workflow_dispatch:
+on: workflow_dispatch
```

Comment:
We probably want to add a schedule for this -- maybe couple hours after the latest PyTorch nightlies are published ?

Reply:
Perfect sounds good to do that in a follow on